src/functions.rst

```diff
@@ -86,7 +84,8 @@ at the site of a :t:`call expression` or a :t:`method call expression` to
 a pattern.
 
 :dp:`fls_lxzinvqveuqh`
-A :t:`function parameter` is an :t:`irrefutable pattern`.
+The :t:`pattern` of a :t:`function parameter` if it exists shall be
```

Comment:
I don't think that "if it exists" is necessary. If there is no function parameter, then the rule does not apply.